docs/explainer.md

```diff
@@ -160,4 +162,45 @@ let writableStream = tcpSocket.writable;
 tcpSocket.close();
 ```
 
+To simplify security analysis, an API for listening for incoming connections is not yet being proposed.
+
 ## UDP
+
+To simplify security analysis, the initial proposal only supports cases where the web app initiates communication with a remote host.
```

Comment:
Perhaps you can say more about the fact that this API simulates a connection (i.e., how received packets will only be routed if they came from the remote address and port, and other packets will be dropped).

Reply:
Text added